assets/js/selecting.js

```diff
@@ -0,0 +1,61 @@
+function isSSLothable(v) {
+	var m = v.major;
+	var n = v.nver;
+	return (
+		(m == 4 && n == 37) ||
+		(m == 5 && n == 38) ||
+		(m == 6 && n == 39) ||
+		(m == 7 && n == 40) ||
+		(m == 8 && n == 41) ||
+		(m == 9 && n == 42) ||
+		(m == 10 && n == 43) ||
+		(m == 11 && n == 43) ||
+		(m == 12 && n == 44) ||
+		(m == 13 && n == 45)
+	);
+}
+
+function getResultText(v) {
+	if (v.major == 0) {
+		return "Ver. 0.0.0-0 indicates a developer unit. This guide only works with retail units.";
+	}
+	if (v.region == "C") {
+		return "You currently cannot hack your 3DS on this version using the main methods. If you want to hack your console, you have to use ntrboot.";
+	}
+	return undefined;
+}
+
+function getRedirect(v) {
+	if (v.string == "other") {
```

Comment:
honestly i'm not confident that people will understand what the "ver/other" dropdown is. Since anyone with cfw should HYPOTHETICALLY already be covered by the cfw check it would probably be better to remove that dropdown entirely

Reply:
People skip that check, but I see you want more people crying on NH, fine